README.md

```diff
@@ -0,0 +1,85 @@
+<p align="center">
+<img src="logo.png">
+</p>
+
+
+iOS Playbook 📚
+==================================
+
+At Babylon, we firmly believe that **transparency** is a core value that should be present in everything we do. This playbook embodies that value by giving an overview on how we work:
```

Comment:
```suggestion
At Babylon, **transparency** is a core value that we believe should be present in everything we do. This playbook embodies it by giving an overview of how we work:
```

Reply:
Either transparency is a core value or it isn't. If it is, you try/believe to make everything as transparent as possible, if you don't, you don't. Believing in something, IMO, does not correlate on acting on it, so we should state that transparency *is* a value, not a hypothetical.

Reply:
The `of` was suggested by Grammarly, and I think it makes sense.

Reply:
I am sorry, but I don't fully agree with this suggestion, although I am not a native speaker. This sentence was taken from the iOS Interview Repo, as is, without any change from me.

Reply:
As a native speaker, it should be "overview *of*". As for the other change...meh. I would be inclined to leave it as it is.

Reply:
👍